script/seed.js

```diff
@@ -1,58 +1,56 @@
-'use strict'
+const db = require('../server/db/db')
+const Product = require('../server/db/models/Product')
 
-const {db, models: {User} } = require('../server/db')
+const products = [{
+  name: 'Blue Hickory',
+  description: 'Here’s our second time working with hickory! This gorgeous bowl has been an eye catcher from the moment we poured it, it’s beauty is just out of this world!',
+  material: 'Hickory',
+  epoxyColor: 'blue',
+  price: 200.00,
+  category: 'Bowl',
+  image: 'https://scontent-mia3-1.xx.fbcdn.net/v/t39.30808-6/276123626_158077886599088_4538128186652999978_n.jpg?_nc_cat=104&ccb=1-5&_nc_sid=8bfeb9&_nc_ohc=N0tLt3VXmlEAX9KWbmJ&_nc_ht=scontent-mia3-1.xx&oh=00_AT9sX3xrEBJ8b8LcArmdNUkEs3z7vHDEfJFcAb386oHKuA&oe=6264C172',
+  customizable: true
+}, {
+  name: 'Jewelry box',
+  description: 'Get your own custom stained box, the perfect gift idea for any event your attending!Our first time working with hickory, and it definitely won’t be our last! We hope you love this small beautiful bowl, perfect for whatever you need, as much as we do!',
```

Comment:
Small typo - "any event you're attending" It also seems that Hickory bowl description was accidentally pasted onto the end of this